Modules/_io/textio.c

```diff
@@ -2427,14 +2427,25 @@ _textiowrapper_encoder_setstate(textio *self, cookie_type *cookie)
 
 /*[clinic input]
 _io.TextIOWrapper.seek
-    cookie as cookieObj: object
-    whence: int = 0
+    offset as cookieObj: object
+    whence: int(c_default='0') = io.SEEK_SET
     /
+
+Change the file position and return the new absolute position.
+
+Offsets relative to the start of the file are usually zero or positive.
```

Comment:
For ``TextIOWrapper`` only the opaque int returned by ``tell()`` is valid -- would that ever be negative?